sroka/api/google_ad_manager/README.md

```diff
@@ -85,3 +85,29 @@ dimensions = ['id', 'name']
 data = get_users_from_admanager(query, dimensions, network_code=1234)
 
 ```
+
+### `get_companies_from_admanager(query, dimensions, network_code)`
+
+#### Arguments
+
+
+* string `query` - obligatory (does not require WHERE clause)
+* list `dimensions` - obligatory
+* int `network_code` - default value taken from config.ini file. If the same service account has access to more than one network, the default value can be overwritten with this argument.
+
+#### Returns
+
+* pandas.DataFrame
```

Comment:
I know that this not only applies to this function, but could we specify when nothing is returned?Could you please add this info to all GAM functions that applies?

Reply:
I just realised that this is always pandas.DataFrame (with data or empty). Do you think it is still worth clarifying?

Reply:
Is it: https://github.com/Wikia/sroka/blob/gam_companies/sroka/api/google_ad_manager/gam_api.py#L264  :-) ?